src/components/header/Header.js

```diff
@@ -7,25 +7,38 @@ import { Link } from 'react-router-dom';
 
 const Header = () => {
   const { user } = useAuthContext();
-  const [searchInput,setSearchInput] = useState("")
-  console.log(searchInput)
+  const [searchInput, setSearchInput] = useState("")
+  const [categoryMenu, setCategory] = useState(false)
+
 
   return (
     <>
       <nav>
         <div className="logo"><a href="/"><img src="https://via.placeholder.com/300x80" alt="logo" /></a></div>
         <div className="searchBar">
-          <input className="searchInput" type="text" placeholder="search item...." 
-             onChange={(e)=>setSearchInput(e.target.value)}></input>
-          <Link to={"/search/" + searchInput }><ImSearch className="searchBtn" color={"white"}/></Link>
-          </div> 
+          <input className="searchInput" type="text" placeholder="search item...."
+            onChange={(e) => setSearchInput(e.target.value)}></input>
+          <Link to={"/search/" + searchInput} onClick={()=>setSearchInput("")}><ImSearch className="searchBtn" size={20} color={"white"} /></Link>
+        </div>
         <div className="iconWrap">
           <ul>
-            <li><Link to={""}><FaBookOpen className="navIcon" /></Link><p>Catagory</p></li>
+            <li onMouseEnter={() => setCategory(true)}>
+              <FaBookOpen className="navIcon" /><p>Category</p>
+            </li>
             <li><Link to={"/cart"}><FaShoppingCart className="navIcon" /></Link><p>Cart</p></li>
             <li><Link to={"/account/"}><FaUserCircle className="navIcon" /></Link>
-            <p>{user === null?"Account" : user.providerData[0].displayName}</p></li>
+              <p>{user === null ? "Account" : user.providerData[0].displayName}</p></li>
           </ul>
+          {categoryMenu === true &&
+            <div className="categoryDropdown" onMouseLeave={() => setCategory(false)}>
+              <ul>
+                <Link to={"/category/men%27s%20clothing"} style={{ textDecoration: 'none',backgroundColor:'orange' }} name={"Men's clothing"}><li>Men's clothing</li></Link>
+                <Link to={"/category/women%27s%20clothing"} style={{ textDecoration: 'none' }} name={"Women's clothing"}><li>Women's clothing</li></Link>
```

Comment:
Can you change these two categories to only 'clothes' later? Because these two will be integrated into one category.

Reply:
Sure, Thanks for review